### PR TITLE
Add option to hide the toolbar when there's one or less items

### LINKED ISSUE #23: Add option to hide the toolbar when there's only one pane
I personally don't really see the point of having a toolbar if there's only one preference pane and would like to hide it. I would still want to use this module though, so it's easy to add additional panes in the future.

So would be nice with a `showToolbarForSinglePane` option.

@DivineDominion Thoughts?

### PR DESCRIPTION
**Pull request by DivineDominion**
Closes #23

### CODE REVIEW

File: Sources/Preferences/PreferencesTabViewController.swift [hunk 1]
```diff
@@ -31,17 +34,15 @@ final class PreferencesTabViewController: NSViewController, PreferencesStyleCont
...
-
-	private func changePreferencesStyleController(preferences: [PreferencePane], style: PreferencesStyle) {
 		let toolbar = NSToolbar(identifier: "PreferencesToolbar")
 		toolbar.allowsUserCustomization = false
 		toolbar.displayMode = .iconAndLabel
 		toolbar.showsBaselineSeparator = true
 		toolbar.delegate = self
 
-		switch style {
+		let preferences = self.preferencePanes
```
Comment (sindresorhus) [inline Sources/Preferences/PreferencesTabViewController.swift]: I would just use `self.preferencePanes` or rather `preferencePanes` directly in the `SegmentedControlStyleViewController`. Maybe also rename it's argument from `preferences` to `preferencePanes`.

File: readme.md [hunk 2]
```diff
@@ -212,7 +213,7 @@ The `PreferencesWindowController` adheres to the [Apple HIG](https://developer.a
 - [DockProgress](https://github.com/sindresorhus/DockProgress) - Show progress in your app's Dock icon
 - [More…](https://github.com/search?q=user%3Asindresorhus+language%3Aswift)
 
-You might also like my [apps](https://sindresorhus.com/apps).
+You might also like Sindre's [apps](https://sindresorhus.com/apps).
```
Comment (sindresorhus) [inline readme.md]: You should add a link to your apps too.

Something like:

> You might also like [Sindre's]() and [Christian's]() apps.

**Comment by sindresorhus** [comment]
When the toolbar is hidden, the window should have a title when using `.segmentedControl` style too.

Now it does not:

Screen Shot 2019-04-16 at 18 44 11

File: readme.md (hunk 2 above)
Comment (DivineDominion) [reply readme.md]: Will do later with another PR for a list of my apps.

**Pull request merged**